torch/nn/modules/module.py

```diff
@@ -335,6 +335,7 @@ def _tracing_name(self, tracing_state):
         return None
 
     def _slow_forward(self, *input, **kwargs):
+        import torch.jit
         input_vars = tuple(torch.autograd.function._iter_variables(input))
         tracing_state = torch.jit.get_tracing_state(input_vars)
```

Comment:
so, I don't know precisely how `torch.jit` was being implicitly imported before, but it's no longer the case, so we have to add an explicit import